lib/utils.js

```diff
@@ -128,6 +135,46 @@ export function parseOutput(lines, inputFilename, pathPrefix) {
     return result;
 }
 
+/***
+ *
+ * @param lines
+ * @param inputFilename
+ * @param pathPrefix
+ * @returns {lineObj[]}
+ */
+export function parseRustOutput(lines, inputFilename, pathPrefix) {
+    const re = /^ --> <source>[(:](\d+)(:?,?(\d+):?)?[):]*\s*(.*)/;
```

Comment:
If anyone a bit more experienced than me with Rust output could tell me how much of the optional stuff here the compiler does not emit, it would simplify the regex a bit :)